### PR TITLE
Add issue template for proposing a talk

### PR DESCRIPTION
**Pull request by bhavin192**
- Should make the process of submitting a new talk very easy

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/propose-a-talk.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<!-- Containers, Hardware. -->
+
+**Title of the talk**
+<!-- Please include a short title -->
+
+**Description**
+<!-- Please include brief abstract about your talk. -->
+
+**Prerequisites**
+<!-- If you have any specific requirement from audience for the talk i.e. software, tools set up on machine etc, please include it here -->
```
Comment (digitronik) [inline .github/ISSUE_TEMPLATE/propose-a-talk.md]: @bhavin192 I think these commented instruction should be visible... maybe like `replace this line` or just in `italic`. what you think?

Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: No, these are meant to be kind of instructions for the person who us creating the issue. Those should not be part of the actual issue that's why those are comments.

Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: Those will be visible to the person who is filing the issue, so it serves the purpose I would say

File: .github/ISSUE_TEMPLATE/propose-a-talk.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a talk
+about: Propose a talk or workshop for upcoming meetup/event
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- You can submit any talks which are related to Python Programming -->
```
Comment (akshay196) [inline .github/ISSUE_TEMPLATE/propose-a-talk.md]: @bhavin192 How about this change?

```diff
- <!-- You can submit any talks which are related to Python Programming -->
+ <!-- You can submit any talks which are, but not limited to, Python Programming -->
```

File: .github/ISSUE_TEMPLATE/propose-a-talk.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a talk
+about: Propose a talk or workshop for upcoming meetup/event
+title: ''
+labels: ''
```
Comment (akshay196) [inline .github/ISSUE_TEMPLATE/propose-a-talk.md]: How about we define a tag for proposals? something like `talk-proposal`

File: .github/ISSUE_TEMPLATE/propose-a-talk.md (hunk 2 above)
Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: Yes, this totally makes sense, thanks a lot for taking look at that

File: .github/ISSUE_TEMPLATE/propose-a-talk.md (hunk 3 above)
Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: Yes, we should have that label, `talk-proposal`

Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: I have renamed the existing `new talk` label to `talk-proposal`

File: .github/ISSUE_TEMPLATE/propose-a-talk.md (hunk 1 above)
Comment (digitronik) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: if those will visible to issue creator then it's ok... I'm just suspect that instruction will not visible.

File: .github/ISSUE_TEMPLATE/propose-a-talk.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a talk
```
Comment (chkumar246) [inline .github/ISSUE_TEMPLATE/propose-a-talk.md]: It should be 'Propose a talk/workshop', as we are mentioning about both in about section.

File: .github/ISSUE_TEMPLATE/propose-a-talk.md (hunk 3 above)
Comment (chkumar246) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: Can we also include level: Beginner, Intermediate, Advance?

File: .github/ISSUE_TEMPLATE/propose-a-talk.md (hunk 1 above)
Comment (chkumar246) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: I think Prerequisites should be given in check list format?

File: .github/ISSUE_TEMPLATE/propose-a-talk.md (hunk 2 above)
Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: I have updated this section please take a look

File: .github/ISSUE_TEMPLATE/propose-a-talk.md (hunk 3 above)
Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: @chkumar246 I think we should keep it simple for now, we can always reiterate over the template later

**Comment by raukadah** [comment]
Thanks @bhavin192

**Pull request merged**